sport/app/football/feed/Competitions.scala

```diff
@@ -167,6 +167,13 @@ object CompetitionsProvider {
       tableDividers = List(2, 6, 21),
     ),
     Competition("751", "/football/euro-2020-qualifiers", "Euro 2020 qualifying", "Euro 2020 qual.", "Internationals"),
```

Comment:
Do we need to remove the competition as well?

Reply:
Sorry "We could probably get rid of the Euro 2020/21 qualifiers now, but awaiting confirmation of that from CP/sport desk.

"

Reply:
Yeah, I think we probably can - I already removed from the tables ordering as they don't appear there at the moment anyway. You can see the results however, so it's probably safer to let editorial decide.